plugins/module_utils/foreman_helper.py

```diff
@@ -588,7 +588,7 @@ def _entity_spec_helper(spec):
         elif argument_value.get('type') == 'nested_list':
             argument_value['type'] = 'list'
             argument_value['elements'] = 'dict'
-            _, argument_value['options'] = _entity_spec_helper(argument_value.pop('entity_spec'))
+            argument_value['options'] = _entity_spec_helper(argument_value.pop('entity_spec'))[1]
```

Comment:
I am not strictly happy about this one, but using `_entyty_spec_ignored` seemed also weird?

Reply:
You could name it dummy instead of _

Reply:
also meh, but did so now as it feels better